widget/penrose/euclidean.sty

```diff
@@ -0,0 +1,102 @@
+--inspired by https://github.com/penrose/penrose/blob/499cd21f39549abd317976f492b803a09ce2277d/packages/examples/src/geometry-domain/euclidean-teaser.style
+
+Colors {
+    -- Keenan palette
```

Comment:
Similarly here regarding style: let's stick to a single indentation strategy. It looks like 2 spaces might be the default, but whichever you prefer!